src/styles/blocks/cardContainer.scss

```diff
@@ -0,0 +1,173 @@
+.main {
+
+  display: flex;
+  flex-wrap: wrap;
+  width: 1024px;
```

Comment:
checklist 11th item, [LAYOUT] - don't hardcode max-width, your catalog container should work fine for any number of cards, you should use max-width instead, and calculate it based on number of curds, their width and gap between them